src/app/models/evt-models.ts

```diff
@@ -84,3 +84,24 @@ export type Relation = OriginalEncodingData & {
 };
 
 export type Description = GenericElementData[];
+export interface WitnessesList {
+    witnesses: Map<Witness>;
```

Comment:
Could you elaborate on the choice for this structure?

Reply:
Each witness inside `witnesses` is a key-value pair. In this way I can quickly retrieve the data I need, but probably in this case it's better to use `[key: string]: string;`? I can't figure out exactly what is best to use in this case.

Reply:
go for the Map, other choices will be considered later